{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
-__author__ = """{{ cookiecutter.full_name }}"""
```

Comment:
Appears this is intentional based on the test failures. Will revert this change.

ref: https://travis-ci.org/nanshe-org/nanshe-cookiecutter/builds/209154268
ref: https://ci.appveyor.com/project/jakirkham/nanshe-cookiecutter/build/1.0.73